addon/mixins/sortable-item.js

```diff
@@ -211,17 +263,35 @@ export default Mixin.create({
     event.preventDefault();
     event.stopPropagation();
 
-    if (this.get('isBusy')) { return; }
+    let drag; 
 
-    let dragOrigin = getY(event);
-    let elementOrigin = this.get('y');
+    if (this.get('group.direction') === 'y') {
+      if (this.get('isBusy')) { return; }
```

Comment:
This guard clause applies equally in both cases. Let’s keep it up-top above the two conditionals.